util/raclgen/lib.py

```diff
@@ -22,13 +25,17 @@
 }
 
 
-def parse_racl_config(config_path: str) -> Dict[str, object]:
+def _read_hjson(filename: str) -> Dict[str, object]:
     try:
-        with open(config_path, 'r') as f_racl_config:
-            racl_config = hjson.load(f_racl_config)
+        with open(filename, 'r') as f_racl_config:
+            return hjson.load(f_racl_config)
     except OSError:
```

Comment:
Please also add "except ValueError", log.error so the error is printed, and raise SystemExit, unless  you are okay with a ValueError generating a backtrace.